paper-dropdown.js

```diff
@@ -277,6 +278,29 @@ class PaperDropDown extends Polymer.mixinBehaviors([PaperDropdownBehavior], Poly
         };
     }
 
+    /**
+     * Checks if item satisfies the filter condition.
+     * If it satisfies and has to be shown to the user,
+     * true is returned, else false is returned.
+     *
+     * Override this method to implement your own custom filter
+     * condition.
+     *
+     * @param searchText Text user entered in search field
+     * @param item Current Item
+     * @return {boolean}
+     * @protected
+     */
+    _filterCheck(searchText, item) {
+        const currentValue = this._getItemLabel(item);
+        if (searchText === "" || currentValue === "") {
+            return true;
+        }
+
+        const re = new RegExp(searchText, "gi");
+        return (re.exec(currentValue) != null);
+    }
+
```

Comment:
You didn't even add `_getValidatity` which means this PR is not tested for form validation which is very essential for any input component.